drivers/clock_control/Kconfig.mcux_lpc

```diff
@@ -0,0 +1,10 @@
+# MCUXpresso SDK CCM
```

Comment:
`CCM` meaning?

Reply:
Thanks for catching this, I have fixed it.